cmpctircd/Configuration/OperatorElement.cs

```diff
@@ -1,49 +1,36 @@
 ﻿using System;
-using System.Configuration;
-using System.Xml;
 using System.Collections.Generic;
-using System.Linq;
 using System.ComponentModel;
+using System.Configuration;
 using System.Globalization;
+using System.Linq;
 
 namespace cmpctircd.Configuration {
     public class OperatorElement : ConfigurationElement {
-        [ConfigurationProperty("name", IsRequired = true, IsKey = true)]
-        public string Name {
-            get { return (string)this["name"]; }
-            set { this["name"] = value; }
-        }
+        public string Name { get; set; }
 
-        [TypeConverter(typeof(HexadecimalConverter))]
-        [ConfigurationProperty("password", IsRequired = true)]
-        public byte[] Password {
-            get { return (byte[]) this["password"]; }
-            set { this["password"] = value; }
-        }
+        public string Password { get; set; }
 
-        [TypeConverter(typeof(TypeNameConverter))]
-        [ConfigurationProperty("algorithm", IsRequired = true)]
-        public Type Algorithm {
-            get { return this["algorithm"] as Type; }
-            set { this["algorithm"] = value; }
-        }
+        public string Algorithm { get; set; }
 
-        [ConfigurationProperty("tls", IsRequired = false, DefaultValue = false)]
-        public bool Tls {
-            get { return (bool) this["tls"]; }
-            set { this["tls"] = value; }
-        }
+        public bool Tls { get; set; }
+        public List<string> Hosts { get; set; }
+        public Type AlgorithmType => Type.GetType(Algorithm);
+        public byte[] PasswordArray => ConvertPassword(Password);
```

Comment:
This was how I decided to solve the converters no longer being respect

(I also need to delete the converters from this file!)